projects/tuist/fixtures/app_with_spm_dependencies/App/Project.swift

```diff
@@ -34,6 +34,7 @@ let project = Project(
                 .external(name: "FirebaseDatabase"),
                 .external(name: "FirebaseFirestore"),
                 .external(name: "IterableSDK"),
+                .external(name: "StripeCore"),
```

Comment:
`StripeCore` is not a package product. It's an internal target, which is the reason for the failing unit tests